docs/pages/versions/unversioned/sdk/calendar.md

```diff
@@ -606,29 +606,29 @@ A source account that owns a particular calendar. Expo apps will typically not n
 
 ### `Calendar.DayOfTheWeek`
 
-| DayOfTheWeek                 | Value |
-| ---------------------------- | ----- |
-| `DayOfTheWeek.Sunday`        | 1     |
-| `DayOfTheWeek.Monday`        | 2     |
-| `DayOfTheWeek.Tuesday`       | 3     |
-| `DayOfTheWeek.Wednesday`     | 4     |
-| `DayOfTheWeek.Thursday`      | 5     |
-| `DayOfTheWeek.Friday`        | 6     |
-| `DayOfTheWeek.Saturday`      | 7     |
+| DayOfTheWeek             | Value |
+| ------------------------ | ----- |
+| `DayOfTheWeek.Sunday`    | 1     |
+| `DayOfTheWeek.Monday`    | 2     |
+| `DayOfTheWeek.Tuesday`   | 3     |
+| `DayOfTheWeek.Wednesday` | 4     |
+| `DayOfTheWeek.Thursday`  | 5     |
+| `DayOfTheWeek.Friday`    | 6     |
+| `DayOfTheWeek.Saturday`  | 7     |
 
 ### `Calendar.MonthOfTheYear`
 
-| MonthOfTheYear                      | Value |
-| ----------------------------------- | ----- |
-| `MonthOfTheYear.January`            |   1   |
-| `MonthOfTheYear.February`           |   2   |
-| `MonthOfTheYear.March`              |   3   |
-| `MonthOfTheYear.April`              |   4   |
-| `MonthOfTheYear.May`                |   5   |
-| `MonthOfTheYear.June`               |   6   |
-| `MonthOfTheYear.July`               |   7   |
-| `MonthOfTheYear.August`             |   8   |
-| `MonthOfTheYear.September`          |   9   |
-| `MonthOfTheYear.October`            |   10  |
-| `MonthOfTheYear.November`           |   11  |
-| `MonthOfTheYear.December`           |   12  |
+| MonthOfTheYear             | Value |
+| -------------------------- | ----- |
```

Comment:
tip: if you want to center these numbers horizontally, you can use sth like this:
```suggestion
| -------------------------- |:-----:|
```